docs/config-file/v2.rst

```diff
@@ -149,24 +159,6 @@ Configuration of the Python environment to be used.
          path: another/package
      system_packages: true
 
-python.version
```

Comment:
Not sure about removing mention of python.version here, maybe add a note that this is a deprecated option now? (it could be a note in the section above)

Reply:
As it's incompatible with `build.os`, which we promoted to the top, I think it makes sense to move it to the "legacy" section below

Reply:
yeah, but I think we should put a note like

```
warning: the python.version option is now deprecated, see ...
```

otherwise people would be looking for the python.version option under the python section and be confused that it disappeared.

Reply:
I am not sure. What users would look for that option?

- The ones that already have a config working, know trivially how to change it (tweak the number in `.readthedocs.yaml`)
- The ones that are looking for a way to configure it, we want them to use the new method, rather than the old one

am I right?

Reply:
There is the option for users looking to add something new under the `python` option (like a requirement, or just re-read what each option does) and realize that `python.version` isn't longer listed there. They will find it eventually, but I've been there were I remember something used to be there and then one day it isn't and had to re-read the whole thing to know what happened :x

Reply:
Added a section with a deprecation warning and a link to the actual docs:

![Screenshot 2021-10-04 at 22-00-48 Configuration File V2 — Read the Docs 5 24 0 documentation](https://user-images.githubusercontent.com/316517/135916665-7d9aa42c-dbc1-4efd-95de-c81e68cb2b30.png)